18F/identity-design-system
# Add "centered" banner component variant

## 🛠 Summary of changes

Formalizes the "centered" variant of the banner component used in the IdP, and adds documentation for banner component usage.

This also includes revisions and recommendations that build upon the [default USWDS documentation](https://designsystem.digital.gov/components/banner/), slated for upstream pull requests:

- Add `width` and `height` attributes to all images ([related blog post](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/), avoids common automated scanner issues)
- Use `us_flag.svg` (vector) instead of `us_flag_small.png` (raster) for improved scalability, smaller file size
- Replaces inline SVG element with reference to `icon.svg` as `<img />` element

## 📜 Testing Plan

1. `npm start`
2. Visit http://localhost:4000/banner/
3. Verify banner appears as expected, both "Default" and "Centered" variants, including recommended markup

## 👀 Screenshots

TBD
